src/service/packet_router.proto

```diff
@@ -42,7 +42,7 @@ message packet_router_packet_up_v1 {
   float snr = 6;
   region region = 7;
   uint64 hold_time = 8;
-  bytes gateway = 9;
+  bytes session_key = 9;
```

Comment:
is this right?

Reply:
Fixed in [85ac774](https://github.com/helium/proto/pull/359/commits/85ac77448b3d26c74a259a80583ec0f1d92a0b49)